### PR TITLE
Bump dependency versions

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Patternfly  3.26.1 ->  3.26.6
Angular Patternfly 4.3.0 -> 4.5.6
origin-web-common 0.0.48 -> 0.0.49
origin-web-catalog 0.0.39 -> 0.0.40

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -13,12 +13,12 @@
     "angular-touch": "1.5.11",
     "angular-route": "1.5.11",
     "angular-bootstrap": "0.14.3",
-    "angular-patternfly": "4.3.0",
+    "angular-patternfly": "4.5.4",
```
Comment (spadgett) [inline bower.json]: Should we take 4.5.6 now that it's out?

Comment (jeff-phillips-18) [reply bower.json]: Yeah, we probably should to get the empty state h1 issue fix.

**Comment by jeff-phillips-18** [comment]
Bumped angular-patternfly to 4.5.6

**Comment by spadgett** [comment]
Thanks @jeff-phillips-18 

[merge]

**Comment by spadgett** [comment]
Flake #1937

**Comment by spadgett** [comment]
[merge]

**Pull request merged**